frontend/src/scenes/settings/SettingsMap.tsx

```diff
@@ -400,6 +401,12 @@ export const SETTINGS_MAP: SettingSection[] = [
                 title: 'Delete environment',
                 component: <TeamDangerZone />,
             },
+
+            {
+                id: 'project-move',
+                title: 'Move environment',
+                component: <ProjectMove />,
+            },
```

Comment:
Proposing we put "Move" last, so that the _destructive_ operation is still very last in the list of settings. Moving is definitely a danger zone operation, but not exactly destructive